src/mixins/FaceCoverMixin.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @copyright Copyright (c) 2022 Marcel Klehr <mklehr@gmx.net>
+ *
+ * @author Marcel Klehr <mklehr@gmx.net>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { mapGetters } from 'vuex'
+import he from 'he'
+
+export default {
+	name: 'FaceCoverMixin',
```

Comment:
If this is used only in `FaceCover`, then it might not be necessary to split into a mixin.

Reply:
It's also used in FaceMergeForm. Might not be worth using a mixin for this but just import the functions, though